airbyte-metrics/metrics-lib/src/main/java/io/airbyte/metrics/lib/MetricClientFactory.java

```diff
@@ -67,6 +71,45 @@ public static synchronized void initialize(final MetricEmittingApp metricEmittin
     }
   }
 
+  private static StatsdConfig getDatadogStatsDConfig() {
+    return new StatsdConfig() {
+
+      /**
+       * @return
+       */
+      @Override
+      public String host() {
+        return configs.getDDAgentHost();
+      }
+
+      /**
+       * @param key Key to lookup in the config.
+       * @return
+       */
+      @Override
+      public String get(String key) {
+        return null;
+      }
+
+    };
+  }
+
+  /**
+   *
+   * Returns a meter registry to be consumed by temporal configs.
+   *
+   */
+  public static MeterRegistry getMeterRegistry() {
+
+    if (DATADOG_METRIC_CLIENT.equals(configs.getMetricClient())) {
+      StatsdConfig config = getDatadogStatsDConfig();
+      return new StatsdMeterRegistry(config, Clock.SYSTEM);
+    }
+
+    // We do not support open telemetry yet.
+    return null;
```

Comment:
is this `null` for OSS cases where temporal isn't configured to send metrics?

Reply:
Yes, or if they want to use otel - in that case we need to configure a different type of config (not StatsdConfig) to support that. If returning null, when starting temporal it will not be instructed to do any metric related things

Reply:
Got it. Can we make that clear in comments?

Reply:
Sure!